### PR TITLE
feat(fa): Adding about form page for applicant

### PR DESCRIPTION
**Pull request by MargretFinnboga**
# ...

https://app.asana.com/0/1201600280110882/1201479524207395

## What

 - about form added to application 

## Why

 - its needed

## Screenshots / Gifs

Screenshot 2022-01-06 at 16 00 54
/ Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      <Accordion singleExpand>
+        <AccordionItem
+          id="id_1"
+          label={formatMessage(aboutForm.personalInformation.accordionTitle)}
+        >
+          <DescriptionText
+            text={aboutForm.personalInformation.accordionAbout}
+            format={{
+              homePageName: 'slóð sveitarfélags',
+              homePageNameUrl: 'https://www.mbl.is/frettir/',
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx:35]: you sure about this url? 😂

File: libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx:1]: About the naming of this file, then I am not a huge fan of naming all files `index.ts`, I would rather like to have it `AboutForm/AboutForm.tsx`. What do you think?

File: libs/application/templates/financial-aid/src/lib/application-templates-financial-aid.tsx [hunk 3]
```diff
@@ -6,12 +6,17 @@ import {
...
+
 import { Roles, ApplicationStates } from './constants'
+
+import * as z from 'zod'
 import { application } from './messages'
 import { dataSchema } from './dataSchema'
 
 type Events = { type: DefaultEvents.SUBMIT }
 
+const applicationName = 'Umsókn um fjárhagsaðstoð'
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/lib/application-templates-financial-aid.tsx:18]: We are not using this and should remove it

File: libs/application/templates/financial-aid/src/lib/application-templates-financial-aid.tsx [hunk 4]
```diff
@@ -6,12 +6,17 @@ import {
   DefaultEvents,
   ApplicationConfigurations,
 } from '@island.is/application/core'
+
 import { Roles, ApplicationStates } from './constants'
+
+import * as z from 'zod'
```
Comment (Rikkiabb) [inline libs/application/templates/financial-aid/src/lib/application-templates-financial-aid.tsx:12]: We are not using zod here

**Review by Rikkiabb — CHANGES_REQUESTED** [review]
Nice, couple of things.

File: libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx (hunk 2 above)
Comment (MargretFinnboga) [reply libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx]: I completely agree, update! 💯

File: libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx (hunk 1 above)
Comment (MargretFinnboga) [reply libs/application/templates/financial-aid/src/fields/AboutForm/index.tsx]: ups 🙃 😆

**Review by Rikkiabb — APPROVED** [review]
👏

**Pull request merged**